docs/stencil-docs/developing-further/customizing-invoices.md

```diff
@@ -43,7 +49,7 @@ The invoice that is generated by clicking Print Invoice relies on a template fil
 
 2. Open the file in the text editor of your choice and edit the HTML file to match your desired customization.
 
-3. Use WebDAV to upload your customized HTML file to WebDAV’s <span class="fp">/template/Snippets/ folder</span>. We recommend the freeware [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV#webdav-client) client CyberDuck (Windows and Mac OS), for which your store’s control panel offers a downloadable connection file prefilled with most store credentials.
+3. Use WebDAV to upload your customized HTML file to WebDAV’s <span class="fp">/template/ folder</span>. We recommend the freeware [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV#webdav-client) client CyberDuck (Windows and Mac OS), for which your store’s control panel offers a downloadable connection file prefilled with most store credentials.

```

Comment:
**Suggestion:** Use WebDAV to upload your customized HTML file to WebDAV’s <span class="fp">/template/ folder</span>. We recommend **CyberDuck**, the freeware [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV#webdav-client) client, available for Windows and Mac OS. **Your** store’s control panel offers a downloadable connection file prefilled with most store credentials.